pxteditor/experiments.ts

```diff
@@ -96,6 +96,10 @@ namespace pxt.editor.experiments {
         return all().some(experiment => isEnabled(experiment));
     }
 
+    export function isEnabledByKey(experimentKey: string): boolean {
```

Comment:
You are leaking the internal key representation for experiencents. This should not happen.